exercises/ecn/README.md

```diff
@@ -183,6 +183,9 @@ There are several ways that problems might manifest:
    and the logs show that the queue length was not high enough to set
    the ECN bit.  Then either lower the threshold in the p4 code or
    reduce the link bandwidth in `topology.json`
+5. When running the traffic from `h1` to `h2` at 1 packet/second (send.py), note that this flow only lasts for 30 seconds. Therefore, it is crucial to start the high-rate traffic from h11 to 
```

Comment:
Probably good to add a sentence like "Alternately, use a parameter larger than the recommended 30 for `send.py` to cause it to send packets for a longer time."